.github/workflows/zombienet_polkadot.yml

```
@@ -0,0 +1,77 @@
+name: Zombienet Polkadot
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  merge_group:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  RUN_IN_CONTAINER: 1
+  FF_DISABLE_UMASK_FOR_DOCKER_EXECUTOR: 1
+  LOCAL_DIR: "./polkadot/zombienet-sdk-tests/tests"
+  GHA_CLUSTER_SERVER_ADDR: "https://kubernetes.default:443"
+
+# only run if we have changes in [subtrate, polkadot] directories or this workflow.
+jobs:
+  isdraft:
+    uses: ./.github/workflows/reusable-isdraft.yml
+  preflight:
+    needs: isdraft
+    uses: ./.github/workflows/zombienet-reusable-preflight.yml
+
+  zombienet-polkadot-functional-0009-approval-voting-coalescing:
+    needs: [preflight]
+    if: ${{ needs.preflight.outputs.changes_substrate || needs.preflight.outputs.changes_polkadot }}
+    runs-on: ${{ needs.preflight.outputs.ZOMBIENET_RUNNER }} # NOTE: should be zombienet-arc-runner (without quotes)
+    timeout-minutes: 60
+    container:
+      image: ${{ needs.preflight.outputs.ZOMBIENET_IMAGE }}
+    env:
+      RELAY_IMAGE: "${{ needs.preflight.outputs.TEMP_IMAGES_BASE }}/polkadot-debug:${{ needs.preflight.outputs.DOCKER_IMAGES_VERSION }}"
```

Should it be `RELAY_IMAGE: "..."` or `POLKADOT_IMAGE: "${{ needs.preflight.outputs.TEMP_IMAGES_BASE }}/polkadot-debug:${{ needs.preflight.outputs.DOCKER_IMAGES_VERSION }}"`?

---

Yes, we use `POLKADOT_IMAGE` as env in `get_images_from_env`.
```suggestion
      POLKADOT_IMAGE: "${{ needs.preflight.outputs.TEMP_IMAGES_BASE }}/polkadot-debug:${{ needs.preflight.outputs.DOCKER_IMAGES_VERSION }}"
```